src/index.html

```diff
@@ -8,6 +8,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars">
+      <div class="stars-block stars--0">
```

Comment:
stars--{number} is stars block modifier, they should be added to the same element

```suggestion
    <div class="stars stars--0">
```

Reply:
не совсем понимаю, что здесь требуется....

у меня есть блок "stars" в котором находятся 6 блоков c классами"stars-block" "stars--N", и в каждом из 6 блоков по 5 блоков самими звездочками "stars__star"

тут что-то лишнее?